MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,961 @@
+</td>
+<td>
+DateTime, DateOnly and DateTimeOffset type property and the property with [DataType(DataType.Date)] and [DataType(DataType.DateTime)] attributes.
+</td>
+<td>
+{{'[DatePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/datepicker?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Time
+</td>
+<td>
+DataFormTimeEditor
+</td>
+<td>
+The DateTime, TimeSpan, TimeOnly and DateTimeOffset  property with [DataType(DataType.Time)] attribute.
+</td>
+<td>
+{{'[TimePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/timepicker?view=net-maui-7.0&tabs=windows)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Checkbox
+</td>
+<td>
+DataFormCheckBoxEditor
+</td>
+<td>
+Bool type property.
+</td>
+<td>
+{{'[CheckBox](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/checkbox?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Switch
+</td>
+<td>
+DataFormSwitchEditor
+</td>
+<td>
+Bool type property.
+</td>
+<td>
+{{'[Switch](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/switch?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Picker
+</td>
+<td>
+DataFormPickerEditor
+</td>
+<td>
+Enum and List type property. 
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[Picker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/picker?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+ComboBox
+</td>
+<td>
+DataFormComboBoxEditor
+</td>
+<td>
+Enum and List type property.
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[SfComboBox](https://help.syncfusion.com/maui/combobox)'| markdownify }}
+</td>
+</tr>
+
+<tr>
+<td>
+RadioGroup
+</td>
+<td>
+DataFormRadioGroupEditor
+</td>
+<td>
+Enum and List type property.
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[RadioGroup](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+AutoComplete
+</td>
+<td>
+DataFormAutoCompleteEditor
+</td>
+<td>
+Enum and List type property.
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[SfAutoComplete](https://help.syncfusion.com/maui/autocomplete)'| markdownify }}
+
+</td>
+</tr>
+</table>
+
+## Text editor
+
+In the text editor, the [Entry](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/entry?view=net-maui-7.0) is loaded.
+
+Text editor have Keyboard type change by using `Keyboard` property and you can set maximum allowed length of input by using `MaxLength` property.
+
+## Multiline Text editor
+
+In the `MultilineText` editor, the [Editor](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/editor?view=net-maui-7.0) is loaded.
+
+And `MultilineText` editor height will auto expand/reduce based on the line wraps in editor , which allowing text to be readable without scrolling the editor.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    [DataType(DataType.MultilineText)]
+    public String Address { get; set; }
```

Comment:
public **string** Address => use string not String

Reply:
String to string